core/src/main/scala/org/apache/spark/streaming/eventhubs/EventHubsClientWrapper.scala

```diff
@@ -131,22 +132,27 @@ private[eventhubs] class EventHubsClientWrapper extends Serializable with EventH
     // Create Eventhubs client
     eventhubsClient = EventHubClient.createFromConnectionStringSync(connectionString)
 
+    // Enabling flag to receive runtime info from eventhub client.
+    val receiverOptions = new ReceiverOptions
+    receiverOptions.setReceiverRuntimeMetricEnabled(true)
```

Comment:
Its approx about (26 bytes + 2 Instant java object). Not sure about size of Instant object.

It definitely is a bit expensive operation. Do you have any approximate ETA for this feature using start seq number?